app/controllers/application_controller.rb

```diff
@@ -95,7 +95,7 @@ def filter_params
 
   def oidc
     @oidc ||= OidcClient.new(
-      ENV.fetch("ACCOUNT_MANAGER_URL"),
+      Services.accounts_api,
```

Comment:
Doh! Ignore all my plek stuff then... this does exactly that :+1: 
Would you consider squashing this back after review?

Reply:
Yeah, I've fixed this up into the earlier commits so the "change to use Plek" commit is now gone.